src/main/resources/db/changesets/20210409_normalizecolumnmetadata.yaml

```diff
@@ -0,0 +1,32 @@
+databaseChangeLog:
+  - changeSet:
+      id: normalize_column_metadata
+      # In step 1 we update snapshot columns with an empty type and get their type and array status from the matching dataset column
+      # In step 2 we make sure that all dataset columns have lower case types
+      # In step 3 we make sure that all snapshot columns have lower case types
+      author: dd
```

Comment:
Should be updated to the actual author, not dd

Reply:
oh whoops...my bad